content/index.md

```diff
@@ -17,70 +17,275 @@ footer:
+  {% assign impl_key = implementation.uniqueKey %}
+  {% assign impl_mapping_file = site.data.wcag-act-rules.implementations[impl_key] %}
+  
+  {% if impl_mapping_file.actRuleMapping %}
+    {% for rule_mapping in impl_mapping_file.actRuleMapping %}
+      {% if rule_mapping.ruleId and rule_mapping.consistency == "complete" %}
+        {% assign current_rule_id = rule_mapping.ruleId %}
+        {% assign all_implemented_ids = all_implemented_ids | push: current_rule_id %}
+        
+        {% if implementation.type == "Automated tool" %}
+          {% assign automated_rule_ids = automated_rule_ids | push: current_rule_id %}
+        {% endif %}
+        {% if implementation.type == "Test methodology" %}
+          {% assign manual_rule_ids = manual_rule_ids | push: current_rule_id %}
+        {% endif %}
+        {% if implementation.type == "Semi-automated tool" %}
+          {% assign semiauto_rule_ids = semiauto_rule_ids | push: current_rule_id %}
+        {% endif %}
+        {% if implementation.type == "Linter" %}
+          {% assign linter_rule_ids = linter_rule_ids | push: current_rule_id %}
+        {% endif %}
+      {% endif %}
+    {% endfor %}
+  {% endif %}
+{% endfor %}
 
-<ul>
-{%- for rule in site.data.wcag-act-rules.rules["act-rules"] %}
-  {%- unless rule.permalink contains '/5f99a7' or rule.permalink contains '/6a7281' or rule.permalink contains '/674b10' %}
-  {%- if rule.proposed == false and rule.deprecated != true %}
-    <li><a href="{{ rule.permalink | relative_url }}">{{ rule.title }}</a></li>
-  {%- endif %}
-  {%- endunless %}
-{%- endfor %}
-</ul>
+{% comment %}Remove duplicates from the rule IDs{% endcomment %}
+{% assign automated_rule_ids = automated_rule_ids | uniq %}
+{% assign manual_rule_ids = manual_rule_ids | uniq %}
+{% assign semiauto_rule_ids = semiauto_rule_ids | uniq %}
+{% assign linter_rule_ids = linter_rule_ids | uniq %}
+{% assign all_implemented_ids = all_implemented_ids | uniq %}
 
-## Rules for WAI-ARIA
 
-The ACT Rules in this section directly relate to WAI-ARIA requirements. These rules have been approved by the WAI-ARIA Working Group. They are fully implemented in at least one [test tool or methodology](../implementations/).
+{% include filter-controls.html %}
 
-<ul>
-{%- for rule in site.data.wcag-act-rules.rules["act-rules"] %}
-  {%- if rule.permalink contains '/5f99a7' or rule.permalink contains '/6a7281' or rule.permalink contains '/674b10' %}
-    <li><a href="{{ rule.permalink | relative_url }}">{{ rule.title }}</a></li>
-  {%- endif %}
-{%- endfor %}
-</ul>
+{% for principle in site.data.wcag-complete.principles %}
+{% for guideline in principle.guidelines %}
+<h2 class="guideline-section">Guideline {{ guideline.num }} - {{ guideline.handle }}</h2>
 
-## Proposed Rules for WCAG 2
+{% for sc in guideline.successcriteria %}
+{% if sc.versions contains "2.2" %}
+<div class="sc-item level-{{sc.level | downcase}}{% if forloop.first %} first{% endif %}">
```

Comment:
Did you not make criterions a list to avoid too many lists on the page? I think it should be generally but I also heard an opinion before how many lists on the page can become a nuisance